src/commands/login/list-users-in-group.ts

```diff
@@ -13,21 +14,43 @@ export class ListUsersInGroup extends BaseCommand<typeof ListUsersInGroup> {
     'group-name': Flags.string({
       summary: 'Name of the user group',
     }),
+    'max-items': Flags.integer({
+      summary: "The total number of items to return in the command's output",
+      max: MAX_ITEMS_LIMIT,
+    }),
+    'starting-token': Flags.string({
+      summary: 'A token to specify where to start paginating',
+    }),
   }
 
   public async run(): Promise<GroupUserMappingsList> {
     const { flags } = await this.parse(ListUsersInGroup)
     const promptFlags = await promptRequiredParameters(['group-name'], flags)
     const schema = z.object({
       'group-name': z.string().max(INPUT_LIMIT),
+      'max-items': z.number().optional(),
+      'starting-token': z.string().optional(),
     })
     const validatedFlags = schema.parse(promptFlags)
 
     ux.action.start('Fetching users in the user group')
-    const listGroupUsersOutput = await vpAdapterService.listGroupUsers(validatedFlags['group-name'])
+    const listGroupUsersOutput = await vpAdapterService.listGroupUsers(validatedFlags['group-name'], {
+      limit: validatedFlags['max-items'] ?? MAX_ITEMS_LIMIT,
+      exclusiveStartKey: validatedFlags['starting-token'] ?? undefined,
+    })
     ux.action.stop('Fetched successfully!')
+    const { lastEvaluatedKey, ...rest } = listGroupUsersOutput
+    if (!this.jsonEnabled()) this.logJson({ ...rest, 'starting-token': lastEvaluatedKey })
+
+    if (listGroupUsersOutput.lastEvaluatedKey && (await confirm({ message: 'Do you want to fetch next page?' }))) {
```

Comment:
Check if `flags['no-input']` is set before prompting. And as discussed with Digvijay we may want to use inquirer to jump ahead, behind or exit.